src/main/scala/com.snowplowanalytics.snowplow/scalatracker/emitters/RequestUtils.scala

```diff
@@ -122,6 +155,47 @@ object RequestUtils {
   }
 
   /**
+   * Attempt a POST request once
+   *
+   * @param host collector host
+   * @param payload event map
+   * @param port collector port
+   * @return Whether the request succeeded
+   */
+  def attemptPost(host: String, payload: Seq[Map[String, String]], port: Int = 80): Boolean = {
+    val stm = System.currentTimeMillis().toString
+    val payloadWithStm = payload.map(_ ++ Map("stm" -> stm))
+    val req = constructPostRequest(host, payloadWithStm, port)
+    val future = pipeline(req)
+    val result = Await.ready(future, longTimeout).value.get
+    result match {
+      case Success(s) => s.status.isSuccess   // 404 match Success(_) too
```

Comment:
Is `// 404 match Success(_) too` a bug or a design feature? Clarify comment, e.g. with a `TODO: ` or a `This is by design`